sdk/compute/Microsoft.Azure.Management.Compute/src/Properties/AssemblyInfo.cs

```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Compute Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Compute Resources.")]
 
-[assembly: AssemblyVersion("28.0.0.0")]
-[assembly: AssemblyFileVersion("28.0.0.0")]
+[assembly: AssemblyVersion("28.1.0.0")]
```

Comment:
For minor version update, you don't have to change AssemblyVersion.  Just update AssemblyFileVersion.

Reply:
Fixed